mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -31,13 +30,17 @@ import org.apache.spark.sql.DataFrame
  */
 @Since("1.2.0")
 class RegressionMetrics @Since("1.2.0") (
-    predictionAndObservations: RDD[(Double, Double)]) extends Logging {
+    predictionAndObservations: RDD[(Double, Double)], isUnbiased: Boolean = true)
```

Comment:
I've skimmed through some literature, and see that some authors call the constant in regression as "bias term" but its not very common. So `isUnbiased` may not be right choice for this variable. As @sethah pointed out, `fitIntercept` is not applicable in other regression models. How about `regThroughOrigin`?